src/components/Plot/Plot.sass

```diff
@@ -1,43 +1,56 @@
 $not-allowed-color: rgba(255, 0, 0, 0.5)
 
 .Plot
+  --color-red-danger: rgba(255, 0, 0, 0.5)
+  --color-blue-water: rgba(125, 245, 255, 0.75)
+  --color-brown-fertilize: rgba(125, 56, 0, .75)
+  --color-green-ok: rgba(0, 255, 0, 0.5)
+  --color-yellow: #ff0
```

Comment:
i had to look each of these up so i decided to name them all. i have really grown to like css native variables, but we can use sassy ones instead if you prefer @jeremyckahn

Reply:
🙏  this is not exactly the most powerful example of them, but maybe we can work on it together in the future. i couldn't figure out how to do a `:root` style block in sass (i typically use these in vanilla CSS with CSS modules these days, or SCSS). and the way they are able to be redefined is actually way more powerful than SASS allows.